jdat_notebooks/ifu_optimal/ifu_optimal.ipynb

```diff
@@ -0,0 +1,821 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Baseline: NIRSpec IFU Optimal Point Source Extraction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* _numpy_ for array math\n",
+    "* _scipy_ for gaussian smoothing\n",
+    "* _specutils_ for Spectrum1D data model\n",
+    "* _jdaviz_ : Cubeviz data visualization tool\n",
+    "* _regions_ to read DS9 regions\n",
+    "* _photutils_ to define circular apertures\n",
+    "* _astropy.io_ for reading and writing FITS cubes and images\n",
+    "* _astropy.wcs,u,coordinates_ for defining and reading WCS\n",
+    "* _astropy.visualization_ to construct RGB images\n",
+    "* _astropy.stats_ for sigma_clipping\n",
+    "* _astropy.utils_ for downloading files from URLs\n",
+    "* _matplotlib_ for plotting spectra and images\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from numpy import exp, sqrt, arange, interp  \n",
+    "\n",
+    "import scipy\n",
+    "\n",
+    "import specutils\n",
+    "from specutils import Spectrum1D\n",
+    "\n",
+    "from jdaviz import CubeViz\n",
+    "\n",
```

Comment:
pip install --upgrade jdaviz 

gives the error

ERROR: Packages installed from PyPI cannot depend on packages which are not also hosted on PyPI.
jdaviz depends on glue-astronomy@ git+https://github.com/glue-viz/glue-astronomy.git 

I had to install jdaviz from github

Reply:
Added glue and glue-jupyter to requirements file.

Reply:
Does this fix your installation woes?